docs/Phoenix Walkthrough.md

```diff
@@ -30,7 +30,7 @@ Next we will install distillery in our `mix.exs`
 ```elixir
   defp deps do
     [ ...,
-     {:distillery, "~> MAJ.MIN"},
+     {:distillery, "~> 1.4", runtime: false},
```

Comment:
I would prefer to not maintain versions in the individual docs, at some point I'll probably template the versions and add something to the build to generate the docs with proper versions, but at this point let's just use `MAJ.MIN`. Can you modify this and the `Walkthrough.md` change to just add `runtime: false`?